pkg/api/routes.go

```diff
@@ -1189,7 +1189,7 @@ func (rh *RouteHandler) GetBlobUpload(response http.ResponseWriter, request *htt
 	}
 
 	response.Header().Set("Location", getBlobUploadSessionLocation(request.URL, sessionID))
-	response.Header().Set("Range", fmt.Sprintf("0-%d", size-1))
+	response.Header().Set("Range", fmt.Sprintf("bytes=0-%d", size-1))
```

Comment:
Not sure about this. We made some changes in https://github.com/project-zot/zot/pull/596/files
See that discussions.